docker-compose.zap.yaml

```diff
@@ -12,10 +12,10 @@ services:
       - ./test-output:/home/node/test-output
 
   zap-baseline-scan:
-    image: owasp/zap2docker-stable:2.14.0
-    command: zap-baseline.py -t http://ffc-demo-payment-web:${PORT:-3000} -c config/zap.conf -r test-output/zap-report.html -a -d -I
+    image: softwaresecurityproject/zap-bare:2.14.0
```

Comment:
this image was updated 14 days ago. Latest tag is v2.15.0
https://hub.docker.com/r/softwaresecurityproject/zap-bare/tags